src/serializers/encoder/v11/encode_graph_entities.c

```diff
@@ -310,7 +310,6 @@ void RdbSaveEdges_v11
 	// already set to the next entry to fetch
 	// for previous edge encide or create new one
 	RG_MatrixTupleIter *iter = GraphEncodeContext_GetMatrixTupleIterator(gc->encoding_context);
-	if(!iter) RG_MatrixTupleIter_new(&iter, M);
```

Comment:
Please add a new boolean function, checking if an iterator is attached to a given matrix `A`
if not, simply attach to `M`